chall01/vtran.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python3
+import sys
+
+tittut = {
+    'a': ".-",
+    'b': "-...",
+    'c': "-.-.",
+    'd': "-..",
+    'e': ".",
+    'f': "..-.",
+    'g': "--.",
+    'h': "....",
+    'i': "..",
+    'j': ".---",
+    'k': "-.-",
+    'l': ".-..",
+    'm': "--",
+    'n': "-.",
+    'o': "---",
+    'p': ".--.",
+    'q': "--.-",
+    'r': ".-.",
+    's': "...",
+    't': "-",
+    'u': "..-",
+    'v': "...-",
+    'w': ".--",
+    'x': "-..-",
+    'y': "-.--",
+    'z': "--..",
+    ' ': " "
+}
+
+def tinnitus(str):
+    ret = ""
+    for i in str:
+        try:
+            ret += tittut[i]
+        except KeyError:
+            usage()
+    if ret:
+        print (ret)
+
+def usage():
+    print ('usage: ./vtran.py <a-zA-Z string>')
+    sys.exit(1)
+
+def main():
+    if len (sys.argv) == 2:
+        tinnitus(sys.argv[1].lower())
+    else:
+        usage()
+main()
```

Comment:
A good practice is to protect your python scripts that are meant to be run from running on import.
It's done by:
```python
if __name__ == "__main__":
```
and then your code (or even better would be to put your code to function called main() and call it here)